examples/OwnerHeightExample.js

```diff
@@ -17,7 +17,7 @@ class OwnerExample extends React.Component {
     this.onScroll = this.onScroll.bind(this);
     this.state = {
       dataList: new FakeObjectDataListStore(100),
-      scrollY: 0,
+      pageYOffset: 0,
```

Comment:
what was the issue in this example

Reply:
`this.state.scrollY` doesn't exist. I'm pretty sure it was a typo and that it was meant to be `this.state.pageYOffset` instead.

Once we switch to typescript (atleast for the examples), we can avoid these situations.